libraries/Microsoft.Bot.Builder.AI.QnA/DialogHelper.cs

```diff
@@ -0,0 +1,188 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder.Dialogs;
+using Microsoft.Bot.Schema;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.AI.QnA
+{
+    /// <summary>
+    /// QnAMaker Active Learning Dialog helper class
+    /// </summary>
+    public class DialogHelper
+    {
+        /// <summary>
+        /// QnA Maker Active Learning dialog name
+        /// </summary>
+        public const string ActiveLearningDialogName = "active-learning-dialog";
+
+        /// <summary>
+        /// Gets QnA Maker Active Learning Dialog.
+        /// </summary>
+        public WaterfallDialog QnAMakerActiveLearningDialog { get; }
+        
+        // Define value names for values tracked inside the dialogs.
+        private const string CurrentQuery = "value-current-query";
+        private const string QnAData = "value-qnaData";
+
+        // Dialog Options parameters
+        private const float DefaultThreshold = 0.03F;
+        private const int DefaultTopN = 3;
+
+        // Card parameters
+        private const string CardTitle = "Did you mean:";
```

Comment:
Users should be able to configure these 3 text fields.

Reply:
We decided not to make these values at this point configurable for the user.

Reply:
@trojenguri but how would this work in other languages if we have hardcoded english text? Am I missing something?

@tomlm @vishwacsena  How do you feel about this? Should all text be configurable in your opinion?